src/tools/devtools/release-notes/release-notes-2.25.0-src.md

```diff
@@ -0,0 +1,19 @@
+
+# DevTools 2.25.0 release notes
+
+Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
+
+## General updates
+
+- Improve DevTools tab bar navigation when the list of tabs is long - [#5875](https://github.com/flutter/devtools/pull/5875)
+- Clear registered service methods between app connections - [#5960](https://github.com/flutter/devtools/pull/5960)
+
+## Memory updates
+
+- Add legend for class types - [#5937](https://github.com/flutter/devtools/pull/5937)
+- Enable sampling for Memory > Profile - [#5947](https://github.com/flutter/devtools/pull/5947)
+
```

Comment:
Would be great to include a screenshot for this @polina-c @CoderDake

Reply:
Looks like Polina added one to the PR description here that we can use: https://github.com/flutter/devtools/pull/5947